docs/src/adoc/index.adoc

```diff
@@ -6899,7 +6899,7 @@ link:{jdkdocs}/java/lang/reflect/Type.html[java.lang.reflect.Type^] object used
 to represent generics in Java.
 
 
-==== GenericTypes
+==== Generic Types
```

Comment:
this should be "The GenericTypes helper". It references a specific class. Don't split that.